Library/Homebrew/test/.rubocop.yml

```diff
@@ -0,0 +1,5 @@
+inherit_from: ../../.rubocop.yml
+
+# `formula do` uses nested method definitions
+Style/NestedMethodDefinition:
+  Enabled: false
```

Comment:
Add a newline at the end of this file (ideally configure your editor to do this automagically!)